homepage revamp.

nisarhassan12 opened the pull request:
Fixes #1004 

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1015"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1015"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

nisarhassan12 commented on src/routes/blog/opensource.md:
```diff
@@ -47,9 +47,9 @@ The [SaaS offering of gitpod.io](/pricing) remains the easiest way to streamline
-- [Snapshots](/features#snapshot) (share a reproducible workspace with your team)
-- [Live Share](/features#share) (invite others into your running workspace)
-- [Unlimited Prebuilds](/features#prebuilt) (making ephemeral dev environments possible)
+- [Snapshots](/#snapshot) (share a reproducible workspace with your team)
+- [Live Share](/#share) (invite others into your running workspace)
+- [Unlimited Prebuilds](/#prebuilt) (making ephemeral dev environments possible)
```
@ChristinFrohne since `/features` no longer exists and all the features listed here are not on home. Would it make sense to update the links to point to the relevant docs  ? Thanks

ChristinFrohne commented on src/components/index/get-started/launch-example-workspace.svelte:
```diff
@@ -2,19 +2,14 @@
 
-<!-- <a href="#">More example workspaces ↗</a> -->
+<div class="text-xl">
+  <a href="/docs/quickstart">More example workspaces</a>
```
Could you use a font size of 16px here (the small paragraph style)?

ChristinFrohne replied:
Could you clarify the text sizes we are using? According to the styles of the mockup by MAD it's:
24px for large paragraphs
18px for medium paragraphs
16px for small paragraphs

I noticed that you used 20px elsewhere as well. How does it fit in?

nisarhassan12 replied:
I have addressed this. Thanks for the review.

ChristinFrohne reviewed (commented):
Looks already great! Thanks @nisarhassan12 👏

- [x] the Intel logo looks too big. Could you reduce the size so it aligns with the other logos?

- [ ] on mobile it's unclear which text belongs to which image. Could you reduce the spacing between the text and the correlating image?
<img width="495" alt="Bildschirmfoto 2021-09-23 um 16 50 26" src="https://user-images.githubusercontent.com/46345125/134530437-cc844ad7-deb8-48b7-a1c9-eb8b041a2dee.png">

Looking forward to the screenshot changes as well :)

phimae commented:
Thanks Nisar! +1 to Chrstin's points. Some additional feedback below:

- [ ] Remove: "Our own XXGB dev image takes only xx seconds to load." until [#5769](https://github.com/gitpod-io/gitpod/issues/5769) is adressed
- [ ] Desktop vs Browser Toggle is not visible and on mobile the icon sizes are still the desktop size (guess that's what you mean with WIP)
![image](https://user-images.githubusercontent.com/83109781/134589099-754f5ed5-5e29-4e06-966a-7417c05fd83f.png)
- [ ] Not sure if the PyCharm Logo is still a placeholder but it's bit blurry, maybe we can get a higher resolution there?
- [ ] Social proof logos on iPad / Tablet don't look too great as one lonely logo breaks into a new line 
![image](https://user-images.githubusercontent.com/83109781/134590840-8e475aa5-8c1c-4a5b-8c23-493df5f215bc.png)

nisarhassan12 commented:
> - [ ]  Not sure if the PyCharm Logo is still a placeholder but it's bit blurry, maybe we can get a higher resolution there?
> - [ ] Desktop vs Browser Toggle is not visible and on mobile the icon sizes are still the desktop size (guess that's what you mean with WIP)

Thanks. @phimae for the review. The IDE screenshots section is WIP. I have addressed the other todos you added.

phimae reviewed (commented):
Hi @nisarhassan12 , the IDE screenshot section + toggle looks good. Not important but should the toggle change its appearance once switched @ChristinFrohne?

- [x] I know it's not supposed to be clicked but I noticed that clicking the JetBrains icons gives them a weird outline. Can this be disabled? 
![image](https://user-images.githubusercontent.com/83109781/134819161-8272d326-2b7b-4fb2-b5c2-fd9465187250.png)

- [ ] Reduce friction in the "Get Started" Section - when clicking on either of the Git sign up buttons, can we redirect users already to their sign up pop-up from the chosen Git provider instead routing them to gitpod.io/login where they essentially again have to select their Git provider? 
![image](https://user-images.githubusercontent.com/83109781/134819265-9e107cf4-0dcf-462c-9165-46a74c8149a8.png)

nisarhassan12 commented:
@ChristinFrohne because of the height of the width of the screenshots being different there is a glitch in the layout. Can you provide me with new screenshots that are the same in both height and width including the shadow? Thanks 

https://user-images.githubusercontent.com/46004116/134849998-e2cc5192-fbc3-436f-97ec-f2391d10d8d2.mov

ChristinFrohne commented:
>Not important but should the toggle change its appearance once switched @ChristinFrohne?

I think it's fine the way it is right now. We could consider reiterating later on.

>can we redirect users already to their sign up pop-up from the chosen Git provider instead routing them to gitpod.io/login where they essentially again have to select their Git provider?

Good point. It's great to keep the sign-up steps as short and simple as possible. I think it's required that users agree to our TOS and currently, that's the only page showing the TOS. But this might be something that can be improved. What do you think @gtsiolis 

>Can you provide me with new screenshots that are the same in both height and width including the shadow?

👍 will look into it now @nisarhassan12

phimae reviewed (commented):
btw, the IDE screenshot + the toggle should be visible on the screen. Currently if you see the toggle, the screenshot is not fully visible, independent of the screen size. Should behave like https://github.com/features/codespaces#:~:text=Get%20started-,Desktop,-Browser

![image](https://user-images.githubusercontent.com/83109781/134880584-da82d040-99ee-4ab6-b066-87b31e3567ee.png)

nisarhassan12 commented:
Thanks, @phimae for bringing this up. 

I have scaled down the screenshot. 

> the screenshot is not fully visible, independent of the screen size.

FYI, The Codespaces screenshot also appears to be cut on some screen sizes. 

![image](https://user-images.githubusercontent.com/46004116/134891595-ed548482-59b6-48fd-9884-11db4dbc6a8e.png)

gtsiolis commented:
> It's great to keep the sign-up steps as short and simple as possible.  But this might be something that can be improved. What do you think @gtsiolis

Keeping the sign-up steps as short is possible sounds optimal. 🔝 

> I think it's required that users agree to our TOS and currently, that's the only page showing the TOS.

That's true but since we're currently browse wrap[[1](https://en.wikipedia.org/wiki/Browse_wrap)] for accepting terms of service for the product, we could also include the same footer notice below the _Get started now_ section on the website. 💡 

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2021-09-27 at 2 01 24 PM (2)" src="https://user-images.githubusercontent.com/120486/134896364-0080986f-84b2-40af-890b-d17b197b067b.png"> | <img width="1440" alt="Screenshot 2021-09-27 at 2 00 44 PM (2)" src="https://user-images.githubusercontent.com/120486/134896371-dc56041d-504f-4303-9df7-f63651f238cd.png"> |

If possible, re-using the same flow with the modal for authenticating as with the product could be nice. 💭 

| Auth modal within the product |
|-|
| <img width="1440" alt="Screenshot 2021-09-27 at 1 57 02 PM (2)" src="https://user-images.githubusercontent.com/120486/134896683-5b4d476e-e5d6-4459-b33c-9e6632c225b0.png"> |

🍊 🍊 🍊 🍊 

Thinking out loud, alternatively, or additionally, once we have the ability to open workspaces and create projects by cloning example repositories in the upcoming weeks (see https://github.com/gitpod-io/gitpod/issues/5583) we could re-use the same flow we have within the product, in the upcoming Teams & Projects feature, but allow anonymous users to access and authenticate as well as also benefit from accessing the same onboarding flow. This could mean that eventually we could remove the _Get started now_ section on the website and instead redirect to `/new`. 🍔 Cc @atduarte @svenefftinge @JohannesLandgraf 

| Onboarding flow within the product |
|-|
|  <img width="1440" alt="Screenshot 2021-09-27 at 2 06 17 PM (2)" src="https://user-images.githubusercontent.com/120486/134897188-804b7818-d094-4df7-b2e3-cfa600a80d23.png"> |

Thanks for the ping, @ChristinFrohne! 🏓

phimae commented:
@gtsiolis @ChristinFrohne just checked my initial hypothesis by looking at the flow from users that come from `product cta clicked` and it looks like indeed quite a portion of users don't make it beyond /login. Around 11% of users who clicked to sign in with a Git provider don't actually sign up and 35% of all users who clicked to open an example workspace.
![image](https://user-images.githubusercontent.com/83109781/134905499-4647368c-1d29-4b3f-ad6e-900e3d985101.png)


--> Would love to have the footer included in the "get started" section

ChristinFrohne reviewed (changes requested):
Thanks @nisarhassan12!

Looks great. There are just a few smaller things that still need to be adjusted:

- [x] change the cursor state when hovering over the toggle, so users see that it's clickable (similar to how it's done on https://github.com/features/codespaces)

- [x] both the desktop and browser screenshot don't have round corners
<img width="120" alt="Bildschirmfoto 2021-09-27 um 14 29 50" src="https://user-images.githubusercontent.com/46345125/134908277-2b499f0d-7147-4ba4-840c-da87e0e8ae02.png">

- [x] any reason why you only implemented 4 of the 5 Jetbrains IDEs shown in the mockup? Also is there any thought behind changing the order?

- [x] the white bubbles next to the screenshot should switch from F9F9F9 to FFFFF on hover if they are inactive(see mockup). I don't think that implemented yet
- [x] could you disable the hover state for the active bubble (currently VS Code)
- [x] in this image the shadow is missing 
<img width="526" alt="Bildschirmfoto 2021-09-27 um 14 37 19" src="https://user-images.githubusercontent.com/46345125/134909388-7a97845c-002e-4bf1-b09e-3fa2253234cc.png">

- [x] I just noticed the image next to "where teams code together" had a wrong URL. I updated the image in the mockup. Please replace them

- [x] the cloud image is cut 
<img width="123" alt="Bildschirmfoto 2021-09-27 um 14 40 07" src="https://user-images.githubusercontent.com/46345125/134909778-36b47da6-8c56-42eb-9f9c-bc1f3cb34870.png">

ChristinFrohne reviewed (commented):
Two more comments regarding the mobile view:

- [ ] could you adjust the radius of the bubble? They almost look like circles
<img width="71" alt="Bildschirmfoto 2021-09-27 um 15 50 50" src="https://user-images.githubusercontent.com/46345125/134922671-f773865a-7931-41cb-9cbe-90cb21895045.png">


- [ ] could you add a bit more spacing between each feature section. It can be difficult to see what text belongs to which feature. Adding a bit more spacing would help lots 

<img width="357" alt="Bildschirmfoto 2021-09-27 um 15 56 18" src="https://user-images.githubusercontent.com/46345125/134922830-c4eedc07-b73a-4b64-b4ac-0663288c54e8.png">
?

nisarhassan12 commented:
> - [x]  in this image the shadow is missing

@ChristinFrohne  I have added the image with the shadow but exporting with shadow again introduces the spacing inconsistency issue. We can't use code to add box-shadow to such images(even if were to it would take a lot of time and effort) the box-shadow with code can only be applied to images that require to be applied at the edges of the box and not inside it to individual items. 

> - [ ] I just noticed the image next to "where teams code together" had a wrong URL. I updated the image in the mockup. Please replace them

@ChristinFrohne I tried to export the image without the whitespace and the box-shadow via double-clicking but I'm unable to do so and the exported image introducing the spacing issue.

nisarhassan12 commented:
> - [ ] could you add a bit more spacing between each feature section. It can be difficult to see what text belongs to which feature. Adding a bit more spacing would help lots

We are reusing the same spacing for the majority of sections for the whole site should I update the spacing globally or just for the features? updating just for the features can result in in-consistency.

ChristinFrohne commented:
>Regarding the cloud image being cut the one in the mockup is also cut

it's fixed in Figma 

>I tried to export the image

I sent you the image on Slack

>We are reusing the same spacing for the majority of sections for the whole site should I update the spacing 
globally or just for the features? updating just for the features can result in in-consistency.

You can also update it for the whole site, but just for mobile and stick to the spacing variants suggested by MAD please :)
